R/carehomes.R

```diff
@@ -614,14 +619,15 @@ carehomes_index <- function(info) {
                   react_pos = index[["react_pos"]])
 
   ## Only incidence versions for the likelihood now:
-  index_run <- index_core[c("icu", "general", "deaths_carehomes_inc",
-                            "deaths_comm_inc", "deaths_hosp_inc",
-                            "admitted_inc", "diagnoses_inc",
-                            "sero_pos_1", "sero_pos_2", "sympt_cases_inc",
-                            "sympt_cases_non_variant_inc",
-                            "sympt_cases_over25_inc",
-                            "sympt_cases_non_variant_over25_inc",
-                            "react_pos")]
+  index_run <- c(time = index[["time"]],
```

Comment:
I think this should be added to index_core, unless there's something I have forgotten

Reply:
I put `time` in `index_run` but not `index_core` so it can be used in the compare but not get saved

Reply:
ah, cool - that's what I've forgotten. Perhaps make a note? The now-old comment above index_core mislead me a bit